tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenTestRunnerTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿/* Copyright 2019–present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public abstract class JsonDrivenTestRunnerTest : JsonDrivenCommandTest
+    {
+        protected Func<IServer> GetPinnedServer;
```

Comment:
Replace this funky Func property with:

        protected IClientSessionHandle _session;
        protected readonly ITestRunner _testRunner;